integration/nwo/fabric/topology/core_template.go

```diff
@@ -243,27 +243,36 @@ metrics:
     prefix: {{ ReplaceAll (ToLower Peer.ID) "." "_" }}
 `
 
-const DefaultViewExtensionTemplate = `
+const DefaultFSCFabricExtensionTemplate = `
 fabric:
   enabled: true
   {{ FabricName }}:
     default: {{ DefaultNetwork }}
-    BCCSP:
-      Default: SW
-      SW:
-        Hash: SHA2
-        Security: 256
-        FileKeyStore:
-          KeyStore:
-    mspConfigPath: {{ .ViewNodeMSPDir Peer }}
-    localMspId: {{ (.Organization Peer.Organization).MSPID }}
     mspCacheSize: 500
     msps: {{ range Peer.ExtraIdentities }}
       - id: {{ .ID }}
+        default: {{ .Default }} 
```

Comment:
is this a boolean value? Why adding this here? Can't we simply add `defaultMSP: id` before `msps:`? This will simplify the configuration file and reduces the risk of misconfigurations. e.g., imagine the case when someone sets multiple msps to `default: true` - debugging nightmare :)

Reply:
one also has to remember to set `defaultMSP`, no?
Shall we address this in another PR where we restructure the configuration?

Reply:
I am suggesting the following format:


```yaml
defaultMSP: MSP3
msps:
  - id: MSP1
  - id: MSP2
  - id: MSP3
  - id: MSP4
```

vs

```yaml
msps:
  - id: MSP1
     default: false
  - id: MSP2
     default: false
  - id: MSP3
     default: true
  - id: MSP4
     default: false
```

> Shall we address this in another PR where we restructure the configuration?

Do you think that will be a bigger change set? Since this config option is introduced here, it would be the right PR already to do it. But if there are more changes, let's create a dedicated PR - if you wish.

Reply:
okay, let's do that.

Reply:
I submitted a new commit for this